compiler/testData/codegen/bytecodeText/inlineClasses/passInlineClassesWithSpreadOperatorToVarargs.kt

```diff
@@ -17,8 +21,6 @@ fun test(u1: UInt, u2: UInt, us: Array<UInt>) {
 // 2 INVOKESTATIC UInt\.box
 // 0 INVOKEVIRTUAL UInt.unbox
 
-// 2 CHECKCAST \[LUInt
```

Comment:
@sfs Please customize test for both backend via:
...TEMPLATES ...

// JVM_TEMPLATES
// 2 CHECKCAST \[LUInt

// JVM_IR_TEMPLATES
// 0 CHECKCAST \[LUInt
// Maybe smth else

Reply:
like forInObjectArrayIndices.kt for example

Reply:
Thanks for the pointer, I didn't know about TEMPLATES. Fixed!